docs/customer/payment.md

```diff
@@ -15,6 +15,8 @@ To add a new payment
     + **Description** is the name of the payment.
         If you wish to hide any information on the customer portal, use **#** followed by the text to hide.
         For example, PayPal 87878787744XXX #payment from the demo.
```

Comment:
Keep in new paragraph and add to example  only 'PayPal 87878787744XXX' will be visible on the customer portal, as the '#payment from the demo' part is hidden.

Reply:
!!! Example
            PayPal 87878787744XXX #payment from the demo.
        !!! Note
            Add a **#** symbol for starting a private comment to store private information along with the public information.